### PR TITLE
Updates color vars across our webapps

### PR DESCRIPTION
**Pull request by jessicamarcus**

### CODE REVIEW

**Comment by jessicamarcus** [comment]
I apologize for the sheer number of files this commit touches!

File: modules-js/react-fleet/src/utilities/constants.ts [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
+
+export const OPTIMISTIC_BLUE = OPTIMISTIC_BLUE_DARK;
+export const FREEDOM_RED = FREEDOM_RED_DARK;
+
 export const DEFAULT_TEXT = '#58585b';
 
-export const A11Y_RED = '#a32e26'; // higher contrast ratio; passes AAA
+export const FOCUS_INDICATOR_COLOR = OPTIMISTIC_BLUE_LIGHT;
+export const ERROR_BORDER_COLOR = FREEDOM_RED_LIGHT;
+export const ERROR_TEXT_COLOR = FREEDOM_RED_DARK;
```
Comment (fionawhim) [inline modules-js/react-fleet/src/utilities/constants.ts:32]: Loving these semantic names.

File: modules-js/react-fleet/src/utilities/constants.ts [hunk 2]
```diff
@@ -1,17 +1,38 @@
...
+//
+// DARK should always be used for text on white or GRAY_000.
+export const OPTIMISTIC_BLUE_LIGHT = '#288be4';
+export const OPTIMISTIC_BLUE_DARK = '#1979cc';
+
+export const FREEDOM_RED_LIGHT = '#fb4d42';
+export const FREEDOM_RED_DARK = '#d22d23';
+
+export const OPTIMISTIC_BLUE = OPTIMISTIC_BLUE_DARK;
+export const FREEDOM_RED = FREEDOM_RED_DARK;
```
Comment (fionawhim) [inline modules-js/react-fleet/src/utilities/constants.ts]: Are these still needed? Maybe delete them?

**Review by fionawhim — APPROVED** [review]
LGTM, though delete the old constants if you can.

File: modules-js/react-fleet/src/utilities/constants.ts (hunk 2 above)
Comment (jessicamarcus) [reply modules-js/react-fleet/src/utilities/constants.ts]: Good call, thanks for the catch 💯

**Pull request merged**